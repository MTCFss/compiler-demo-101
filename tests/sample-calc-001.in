1
12;
6; 5; 6 + 2;
5; 3
12 + 2
45 * 6;
1 + 3 * 9;
i = 5;
i = i;
i++
++i
--i
i--
i = 6 + 5
i
5 * i
