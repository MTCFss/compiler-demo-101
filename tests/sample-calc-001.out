1
12
6
5
8
5
3
14
270
28
5
7
6
6
11
55
